Java/README.md

```diff
@@ -1,46 +1,54 @@
+    openjdk version "13.0.2" 2020-01-14
+    OpenJDK Runtime Environment AdoptOpenJDK (build 13.0.2+8)
+    OpenJDK 64-Bit Server VM AdoptOpenJDK (build 13.0.2+8, mixed mode, sharing)
+
+You might also see something like this:
 
     java version "1.8.0_45"
     Java(TM) SE Runtime Environment (build 1.8.0_45-b14)
     Java HotSpot(TM) 64-Bit Server VM (build 25.45-b02, mixed mode)
 
+Which means you have the old Oracle version of Java installed. That is fine, too.
+(Unless you're in a corporate environment; see the note at the end of this page.)
+
 If you don't see the output like above then you need to install Java on your system.
 
 ### Using Homebrew
 
-    brew update
-    brew tap AdoptOpenJDK/openjdk
+    brew install openjdk
 
-You can list the available JDK versions via:
+If you're curious, you can list all the available JDK versions via:
 
     brew search jdk
 
-The current LTS release of Java is JDK 11, which you can install via:
-
-    brew cask install adoptopenjdk11
-
-If you wish to install Java 8, you can do so with:
-
-    brew cask install adoptopenjdk8
-
-Check if Java is correctly installed by running the `java -version` command again.
+There will be a lot of options here! If you are not a serious Java developer, don't worry about them, and just use the default `openjdk`.
 
-### Using the Oracle installer
+Once you've done that, check if Java is correctly installed by running the `java -version` command again.
 
-Please [download the macOS version](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html) as per the instructions on the [Oracle website](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-macos.html#GUID-F575EB4A-70D3-4AB4-A20E-DBE95171AB5F). Do note that this method requires you to login/create a new account with Oracle.
+### Downloading and installing manually
 
-#### Add Java to PATH
+Open a web browser and go to <https://adoptopenjdk.net/>. Select "OpenJDK 11 (LTS)" and "HotSpot".
+Click the big "Latest release" button and run the installer file that gets downloaded.
 
-Add `JAVA_HOME` to your environment variables by adding the line below to your `env.sh` (see [iTerm2](/mac-setup/iTerm/README.html) section if you don't have a `env.sh` file).
+Once you've done that, check if Java is correctly installed by opening a new Terminal session and running the `java -version` command again.
 
-    export JAVA_HOME=`/usr/libexec/java_home -v 11`
+## Important note for corporate users
 
-You should have Java working now. Two popular IDE alternatives for writing Java are [IntelliJ](https://www.jetbrains.com/idea/download/) or [Eclipse](https://www.eclipse.org/downloads/).
+If you are working at a company or an "enterprise", and you choose to install the Oracle JDK instead of the OpenJDK JDK,
+it might cost you money, due to Oracle's new licensing terms as of 2019.
+You should probably consult a licensing expert or lawyer about that.
+Or just use OpenJDK instead, and everything will be fine.
```

Comment:
Many enterprises would actually want the users to use the Oracle version and not the OpenJDK version. I would suggest removing line 52-54

Reply:
I have toned down the language, removed these lines, restored instructions on installing the Oracle JDK, and added a note that some organizations may in fact require it. Please have another look.